python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceText2SpeechTransformerPromptSchema.ts

```diff
@@ -0,0 +1,108 @@
+import { PromptSchema } from "../../utils/promptUtils";
+
+export const HuggingFaceText2SpeechTransformerPromptSchema: PromptSchema = {
+  // See https://huggingface.co/transformers/v4.11.3/main_classes/model.html and
+  // https://github.com/huggingface/transformers/blob/main/src/transformers/modeling_utils.py
+
+  input: {
+    type: "string",
+  },
+  model_settings: {
+    type: "object",
+    properties: {
+      torch_dtype: {
```

Comment:
Do we need a "model" property as well to be able to specify the model ID?

Reply:
It's not currently supported but if we'll be adding it then I can add it here to require one less change later?

Reply:
I can add it too. I was thinking all HF model setting schemas accept a "model". Then the model parsers use it as appropriate.

Reply:
I can add to each in next PR

Reply:
> I was thinking all HF model setting schemas accept a "model". Then the model parsers use it as appropriate.

This also fits in with this general task of separating out the pipeline generation params from the actual completion params: https://github.com/lastmile-ai/aiconfig/issues/824

cc @Ankush-lastmile